include/deal.II/algorithms/timestep_control.h

```diff
@@ -139,8 +139,11 @@ namespace Algorithms
     /**
      * Set size of the first step. This may be overwritten by the time
      * stepping strategy.
+     *
+     * @param[in] step The size of the first step, which may be overwritten by
+     *   the time stepping strategy.
      */
-    void start_step (double);
+    void start_step (double step);
```

Comment:
Yes, nice. Can I ask you for a small addition? In general, we mark all input arguments as `const` (here that makes no difference to the compiler, but it makes a difference to the human reader: a `const` argument must clearly be an input argument). Would you be willing to add the `const` before the `double`?